app/models/upload.rb

```diff
@@ -74,7 +74,7 @@ def ready_for_chunks?
   end
 
   def check_readiness!
-    ready_for_chunks? or raise ConsistencyException, 'Upload is not ready'
+    ready_for_chunks? || raise(ConsistencyException, 'Upload is not ready')
```

Comment:
raise(ConsistencyException, 'Upload is not ready') unless ready_for_chunks?

Reply:
:man_shrugging: :potato: :tomato:

Reply:
I do think it is a little more clear 👓

Reply:
Sooo, that caused a test to fail because the `#check_readiness!` method now returns `nil` instead of a truthy value... I think I'm going to change it back to using `||` now, unless you have any objections.

Reply:
I still think its more readable as a guard clause
```
raise(ConsistencyException, 'Upload is not ready') unless ready_for_chunks?
true
```